tests/test_frame.py

```diff
@@ -23,3 +23,24 @@ def test_frame_expand_date_out_of_range():
 
     assert math.isnan(expanded.at[1, "effectiveDateTime.tz"])
     assert pd.isna(expanded.at[1, "effectiveDateTime.local"])
+
+
+def test_local_and_timezone_split():
+    original = pd.DataFrame(
+        [
+            {"effectiveDateTime": "2020-08-08 11:00:00+0300", "id": "obs1"},
+            {"effectiveDateTime": "2020-08-09 10:00:00-0400", "id": "obs2"},
+        ]
+    )
+
+    expanded = Frame.expand(original)
+
+    assert expanded.at[0, "effectiveDateTime.tz"] == 3.0
+    assert expanded.at[0, "effectiveDateTime.local"] == pd.Timestamp(
+        "2020-08-08 11:00:00", tz="utc"
+    )
```

Comment:
I know it's late to comment on this aspect, but why is effectiveDateTime.local specified as UTC? That seems confusing and incorrect. Is there a specific reason not to set tz=None? I don't have any context here, what is the purpose of splitting a valid Timestamp with timezone into separate timezone and local parts anyway?

Reply:
So, time comparison in pandas works in the most situations when UTC is true (e.g. comparison between columns works when time zone is fixed). Likewise, certain data (such as fasting) is most commonly analyzed in the local time zone rather than UTC because of daylight.

However, I'm definitely open to other ideas or even auto-processing an `effectiveDateTime.utc` column. I just want to make the end result easiest for the researcher to handle. (Obviously, if we add a third column, the size will increase a bit.)

Reply:
We could replace the the `.local` column with a `.utc` column (values converted to UTC) and keep the `.tz` column. This would keep the dataframe's timestamps in the same timezone, but allow users to revert back to local times by applying the `.tz` values.

Reply:
@mjtieman I think I'll go ahead and add a `.utc` column. Since I very recently changed how date columns are parsed, I don't want to pull the rug on removing `.local` so quickly, but I understand your logic. I'll make a new issue.